src/gtk-4.0/widgets/_shortcuts.scss

```diff
@@ -0,0 +1,37 @@
+shortcuts-section {
+    margin: rem(12px);
+}
+
+shortcuts-group > label {
+    @extend .h4;
+}
+
```

Comment:
Okay sorry for this back and forth but it looked wrong with the labels butting up against the keys. I think this looks pretty correct?

```suggestion

// Characters between keys, like + or /
 shortcut .dim-label {
     margin-left: rem(3px);
     margin-right: rem(3px);
 }
 
```

![Screenshot from 2022-01-11 15 12 54](https://user-images.githubusercontent.com/611168/149029667-d9db009a-e616-4ca3-9d11-77ae329d5074.png)